src/content/documents/Inheritance.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Inheritance"
+author: "github: 71573739"
 ---
```

Comment:
Is this valid? As far as I'm aware, front-matter can only be defined and read by Astro in `.mdx` files.

Reply:
I think it works with md as well